cvmfs/network/download.cc

```diff
@@ -1365,6 +1402,81 @@ void DownloadManager::ReleaseCredential(JobInfo *info) {
 }
 
 
+/* Sort links based on the "pri=" parameter */
+static bool sortlinks(const std::string &s1, const std::string s2) {
```

Comment:
The second parameter should likely be a reference too. I'll fix this in a clang-tidy patch